Source/FlowEditor/FlowEditor.Build.cs

```diff
@@ -12,7 +12,8 @@ public FlowEditor(ReadOnlyTargetRules target) : base(target)
 		{
 			"EditorSubsystem",
 			"Flow",
-			"MessageLog"
+			"MessageLog",
+			"AIModule", // For BlueprintNodeHelpers::DescribeProperty (could be copy/pasted out to remove editor-only dependency)
```

Comment:
This is an editor-only dependency on the AIModule, so I didn't think it was a problem.

But it could be removed by copying or otherwise reimplementing the DescribeProperty function that is used.